storage/Devices/StrayBlkDeviceImpl.cc

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2018 SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact Novell, Inc.
+ *
+ * To contact Novell about this file by physical or electronic mail, you may
+ * find current contact information at www.novell.com.
+ */
+
+
+#include "storage/Devices/StrayBlkDeviceImpl.h"
+#include "storage/Devicegraph.h"
+#include "storage/Storage.h"
+#include "storage/Environment.h"
+#include "storage/SystemInfo/SystemInfo.h"
+#include "storage/Utils/Exception.h"
+#include "storage/Utils/StorageDefines.h"
+#include "storage/Utils/XmlFile.h"
+#include "storage/Utils/CallbacksImpl.h"
+#include "storage/Prober.h"
+
+
+namespace storage
+{
+
+    using namespace std;
+
+
+    const char* DeviceTraits<StrayBlkDevice>::classname = "StrayBlkDevice";
+
+
+    StrayBlkDevice::Impl::Impl(const xmlNode* node)
+	: BlkDevice::Impl(node)
+    {
+    }
+
+
+    string
+    StrayBlkDevice::Impl::get_pretty_classname() const
+    {
+	// TRANSLATORS: name of object
```

Comment:
Translators will have a hard time translating our internal abstraction to something meaningful, They will not even know that it is our made-up term. Please give them some hints, maybe copy the class description.

Reply:
Seems difficult without repeating the explanation over and over since the term is already used four times.